bgpd/bgp_table.h

```diff
@@ -372,4 +373,31 @@ static inline void bgp_nexthop_set_node_info(struct bgp_node *node,
 	node->info = bnc;
 }
 
+static inline struct bgp_info *bgp_info_get_node_info(struct bgp_node *node)
+{
+	return node->info;
+}
+
+static inline void bgp_info_set_node_info(struct bgp_node *node,
+					  struct bgp_info *bi)
+{
+	node->info = bi;
+}
+
+static inline struct bgp_table *bgp_table_get_node_info(struct bgp_node *node)
```

Comment:
We are misnaming the function here. bgp_table_get_node_info suggests that from a bgp_table we get back a node (node_info) which is not at all what it's doing. Name should be something like bgp_node_get_table or bgp_table_from_node or bgp_get_table_from_node.